lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -231,7 +231,7 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 			}
 		}
 #if UFE_PREVIEW_VERSION_NUM >= 2015
-		else if (!prim.IsValid())
+		else if (!prim.IsValid() && !InPathChange::inPathChange())
```

Comment:
Don't send the delete when we are in a path change. The UFE rename notif is sent [here](https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/ufe/UsdUndoRenameCommand.cpp#L130)

Reply:
Shame that we didn't have a failing test to report this.  Feels like we're missing code coverage.

Reply:
Indeed, code coverage is high on my wish list. What stops us from adding a test now?

Reply:
How would we even test this?

Reply:
Some kind of rename failure, I would hope.